gluoncv/model_zoo/fcn.py

```diff
@@ -35,9 +35,9 @@ class FCN(SegBaseModel):
     """
     # pylint: disable=arguments-differ
     def __init__(self, nclass, backbone='resnet50', norm_layer=nn.BatchNorm,
-                 aux=True, ctx=cpu(), **kwargs):
+                 aux=True, ctx=cpu(), pretrained=True, **kwargs):
         super(FCN, self).__init__(nclass, aux, backbone, ctx=ctx,
-                                  norm_layer=norm_layer, **kwargs)
+                                  norm_layer=norm_layer, pretrained=pretrained, **kwargs)
```

Comment:
same here. The args will be automatically passed through `**kwargs`

Reply:
This can be passed through `**kwargs`

Reply:
Yes, but I think it is helpful to keep this explicit. When I was looking into the code, I thought this was difficult to understand without being written out explicitly. If you still think it is better off being inside ``**kwargs``, let me know and I will remove it.